app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if current_user.present?  %>
+    <%= render "shared/list_blog" %>
+<% else %>
+    <p>Please login to use the service !</p>
+<% end %>
```

Comment:
<%= current_user.present? ? render("shared/list_blog") : content_tag(:p, "Please login to use the service !") %>